v0.2.1/app-shell/app-shell.html

```diff
@@ -360,6 +360,7 @@
       let manifests = await this._fetchManifestList();
       return await ManifestTools.loadManifest(this._config, this._loader, manifests);
     }
+
```

Comment:
Do we need to include the v0.2.1 change here and below for the extension?

Reply:
I'm not sure I understand. Does my [comment](https://github.com/PolymerLabs/arcs-cdn/pull/46#issuecomment-347344510) answer this at all?

Reply:
Yes, your comment answers, thanks.